lib/puppet/functions/openvpn/file_content.rb

```diff
@@ -0,0 +1,20 @@
+# A function that return a file content at apply time
+Puppet::Functions.create_function(:'openvpn::file_content') do
+  # @param filename Path of file to cat
+  # @return [String] Returns the file content
+
```

Comment:
Does this empty line break puppet-strings?

Reply:
I hope a test covers this.

Reply:
```suggestion

```

Reply:
Puppet-lint [covers this](https://github.com/rodjek/puppet-lint/pull/931) for puppet manifests, but I'm not aware of anything that would have warned you here, (and I'm only guessing it would break).